nixos/modules/services/misc/nixos-manual.nix

```diff
@@ -58,6 +58,15 @@ let
       exec "$browser" ${entry}
     '';
 
+  desktopItem = pkgs.makeDesktopItem {
+    name = "nixos-manual";
+    desktopName = "NixOS Manual";
+    genericName = "View NixOS documentation in a web browser";
+    # TODO: find a better icon (Nix logo + help overlay?)
+    icon = "system-help";
+    exec = "xdg-open ${entry}";
```

Comment:
Yes, I meant `Type=Link`, but I just read the specs now and never used that &ndash; thanks.